tempesta_fw/sock.c

```diff
@@ -1355,6 +1338,9 @@ ss_tx_action(void)
 		}
 dead_sock:
 		sock_put(sk); /* paired with push() calls */
+		while ((skb = ss_skb_dequeue(&sw.skb_list)))
+			kfree_skb(skb);
```

Comment:
Good catch! Now i see why the memory leak appeared for me in situations with heavy failovering.

Reply:
Indeed. Thanks for catching this! I feel much better now.